kategory/src/test/kotlin/kategory/laws/TraverseLaws.kt

```diff
@@ -17,14 +17,14 @@ class TIF {
 }
 
 object TraverseLaws {
+    // FIXME(paco): cf is only required because AP::pure will crash the inliner
```

Comment:
No there's a bug even if you use a lambda. It fails to find and capture the context of the lambda O.O